x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/maintenance_window_callout/maintenance_window_callout.test.tsx

```diff
@@ -54,6 +60,18 @@ describe('MaintenanceWindowCallout', () => {
 
     appToastsMock = useAppToastsMock.create();
     (useAppToasts as jest.Mock).mockReturnValue(appToastsMock);
+    (useKibana as jest.Mock).mockReturnValue({
```

Comment:
nit: extract to mock so it can be imported ala the toasts mock above

``` ts
let appToastsMock: jest.Mocked<ReturnType<typeof useAppToastsMock.create>>;
```